### PR TITLE
Lint patch

### PR DESCRIPTION
**Pull request by aneeshkp**
Updated linter and fixed code as per make lint

### CODE REVIEW

File: .golangci.yaml [hunk 1]
```diff
@@ -5,42 +5,72 @@ run:
...
 
 linters:
   disable-all: true
   enable: # NOTE: please keep this list alphabetically sorted
     - bodyclose
     - deadcode
+    - decorder
     - dogsled
-    - errcheck
+    #- errcheck
```
Comment (jzding) [inline .golangci.yaml:60]: why disable this?

Comment (aneeshkp) [reply .golangci.yaml:60]: it was not working before, but with this fix now turning  on that will generate many fixed where we just ignore the  error.

**Review by jzding — APPROVED** [review]
/lgtm

**Pull request merged**